tests/common/factories/annotation.py

```diff
@@ -26,10 +26,7 @@ class Meta:
     target_uri = factory.Faker("uri")
     text = factory.Faker("paragraph")
     userid = factory.LazyFunction(
-        lambda: "acct:{username}@{authority}".format(
-            username=FAKER.user_name(),  # pylint:disable=no-member
-            authority="localhost",
-        )
+        lambda: f"acct:{FAKER.user_name()}@localhost"  # pylint:disable=no-member
```

Comment:
This one is a little bit of a shame, as the placeholders here actually provide a bit of info about what everything is.